.github/workflows/cmake.yml

```diff
@@ -458,8 +458,7 @@ jobs:
 
       - name: Generate build system
         run: |
-          # TODO: Re-enable default features, once the liblzma package become available to build again.
-          cmake -B build --preset ${{ matrix.conf.preset }} -DWERROR=ON -DVCPKG_MANIFEST_NO_DEFAULT_FEATURES=ON -DVCPKG_MANIFEST_FEATURES="wallet;miniupnpc;zeromq;tests"
+          cmake -B build --preset ${{ matrix.conf.preset }} -DWERROR=ON
 
```

Comment:
Is this change related to the liblzma pinning?

Reply:
After liblzma pinning, it is possible to use all default dependency features, which is the goal of the liblzma pinning.

This commit reverts changes introduced https://github.com/hebasto/bitcoin/pull/117 as a quick response to xz backdoor and the following GitHub repo disabling.